jarbas/core/management/commands/companies.py

```diff
@@ -31,7 +34,7 @@ def save_companies(self):
         skip = ('main_activity', 'secondary_activity')
         keys = tuple(f.name for f in Company._meta.fields if f not in skip)
         with lzma.open(self.path, mode='rt', encoding='utf-8') as file_handler:
-            for row in csv.DictReader(file_handler):
+            for row in map(self.transform,rows.import_from_csv(filename_or_fobj=file_handler)):
```

Comment:
This might work but this is not pythonic. First of all, style:

1. [try to keep things under 80 chars](https://www.youtube.com/watch?v=wf-BqAjZb8M)
1. list comprehension _usually_ are more readable than `map` - Zen of Pytthon
1. always add a space after come when separating values (in sequences) and arguments (in function calls) — also PEP8

I'd try something like:

```
for row in rows.import_from_csv(file_handler):
    row = row._asdict()  # and we can get rid of the self.transform call
    …
```

Reply:
Okay @cuducos Thanks for the review,I would make the necessary changes .